src/components/range-inputs.tsx

```diff
@@ -0,0 +1,211 @@
+import '@isrd-isi-edu/chaise/src/assets/scss/_range-input.scss';
+import { useState, useRef } from 'react';
+import { windowRef } from '@isrd-isi-edu/chaise/src/utils/window-ref';
+import InputSwitch from '@isrd-isi-edu/chaise/src/components/input-switch';
+
+const INTEGER_REGEXP = /^\-?\d+$/;
+
+const FLOAT_REGEXP = /^\-?(\d+)?((\.)?\d+)?$/;
+
+const TIMESTAMP_FORMAT = 'YYYY-MM-DDTHH:mm';
+
+const DATE_FORMAT = 'YYYY-MM-DD';
+
+/**
+ * an object for mapping the error types to error messages
+ */
+const errorMsgMap: {
+    [key: string]: string;
+} = {
+    'empty': 'Please enter either From or To value',
```

Comment:
I don't think this message is necessary since submission is disabled when no value is present.

Reply:
yes I forgot to remove this from the error map even though we don't return this error from the validation function anymore